### PR TITLE
chore: connect landing page v2

### PR DESCRIPTION
**Pull request by easyrun42**
## PR-Codex overview
The focus of this PR is to update images and links in the `ConnectLanding` component on the web3 frontend toolkit page.

### Detailed summary
- Updated images for different sections in the `ConnectLanding` component.
- Changed links to point to specific URLs.
- Adjusted text and descriptions for better clarity.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/pages/connect.tsx [hunk 1]
```diff
@@ -280,7 +279,8 @@ const ConnectLanding: ThirdwebNextPage = () => {
             trackingCategory={TRACKING_CATEGORY}
             ctaLink="https://portal.thirdweb.com/payments"
             gradient="linear(to-r, #4490FF, #4490FF)"
-            lottie={checkoutLottie}
```
Comment (jnsdls) [inline src/pages/connect.tsx:283]: we removed these, did we also delete the files? if not let's do that if they are no longer used

File: src/pages/connect.tsx [hunk 2]
```diff
@@ -168,7 +164,6 @@ const ConnectLanding: ThirdwebNextPage = () => {
             ctaLink="/dashboard/connect/playground"
             contactUsTitle="Book Demo"
             gradient="linear(to-r, #4490FF, #4490FF)"
-            lottie={connectLottie}
```
Comment (jnsdls) [inline src/pages/connect.tsx:171]: we removed these, did we also delete the files? if not let's do that if they are no longer used

**Review by jnsdls — APPROVED** [review]
no-blocking feedback

File: src/pages/connect.tsx (hunk 1 above)
Comment (easyrun42) [reply src/pages/connect.tsx:283]: Yeah we use that in the checkout landing page [here](https://github.com/thirdweb-dev/dashboard/blob/main/src/pages/checkout.tsx#L14)

**Comment by juandolealt** [comment]
@jnsdls i'm gonna make some more changes in here - will ping u when it's ready to merge

**Comment by jnsdls** [comment]
> @jnsdls i'm gonna make some more changes in here - will ping u when it's ready to merge

if it's just copy changes then the approval stands, simply merge it yourself in that case

**Comment by juandolealt** [comment]
@easyrun42 looks like we deleted the hero lottie? can we add that back? 🙏

**Comment by juandolealt** [comment]
@easyrun42 looks like we accidentally deleted the hero lottie - can we add that back? 🙏

**Comment by jnsdls** [comment]
how come we're adding 500k+ lines in this PR?

**Comment by juandolealt** [comment]
> how come we're adding 500k+ lines in this PR?

@jnsdls it's the hero lottie that Mago accidentally removed - did i add that back correctly?

**Comment by jnsdls** [comment]
> > how come we're adding 500k+ lines in this PR?
> 
> @jnsdls it's the hero lottie that Mago accidentally removed - i just reuploaded it, did i add that back correctly?

fixed

**Pull request merged**